docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
```

Comment:
This seems incorrect - "running QtCreator in target environment" seems to imply you would run QtCreator on, for instance, a TX2 or qemu INSIDE PELUX - this isn't what you mean, I'm sure.

Reply:
Yeah that's write i should write in SDK environment i guess.